src/platform/Ameba/DiagnosticDataProviderImpl.cpp

```diff
@@ -143,8 +143,8 @@ CHIP_ERROR DiagnosticDataProviderImpl::GetNetworkInterfaces(NetworkInterface **
                 ifp->type = EMBER_ZCL_INTERFACE_TYPE_ETHERNET;
             else
                 ifp->type = EMBER_ZCL_INTERFACE_TYPE_WI_FI;
-            ifp->offPremiseServicesReachableIPv4 = false;
-            ifp->offPremiseServicesReachableIPv6 = false;
+            ifp->offPremiseServicesReachableIPv4.SetNonNull(false);
+            ifp->offPremiseServicesReachableIPv6.SetNonNull(false);
```

Comment:
This is wrong, per spec.  Filed https://github.com/project-chip/connectedhomeip/issues/14936

Reply:
https://github.com/project-chip/connectedhomeip/pull/15008